src/tea_timer.c

```diff
@@ -85,6 +100,9 @@ static void menu_window_load(Window *window) {
   s_menu_layer = menu_layer_create(bounds);
   menu_layer_set_callbacks(s_menu_layer, NULL, (MenuLayerCallbacks){
     .get_num_rows = get_sections_count_callback,
+  #ifdef PBL_ROUND
+    .get_cell_height = get_cell_height_callback,
```

Comment:
Since this callback is only used on round displays, use `PBL_IF_ROUND_ELSE()` inline here:

```
.get_cell_height = PBL_IF_ROUND_ELSE(get_cell_height_callback, NULL),
```